lib/fizzy/execute.cpp

```diff
@@ -248,23 +248,24 @@ void branch(
 {
     const auto code_offset = read<uint32_t>(immediates);
     const auto imm_offset = read<uint32_t>(immediates);
-    const auto stack_height = static_cast<size_t>(read<uint32_t>(immediates));
+    const auto stack_drop = read<uint32_t>(immediates);
     const auto arity = read<uint8_t>(immediates);
 
     pc = code.instructions.data() + code_offset;
     immediates = code.immediates.data() + imm_offset;
 
     // When branch is taken, additional stack items must be dropped.
-    assert(stack.size() >= stack_height + arity);
+    assert(static_cast<int32_t>(stack_drop) >= 0);
```

Comment:
It was `int` on the parser side
```suggestion
    assert(static_cast<int>(stack_drop) >= 0);
```